src/awakening_stage/gui/SelectBuildingPopup.tscn

```diff
@@ -1,22 +1,23 @@
 [gd_scene load_steps=3 format=3 uid="uid://borrsf8mdual2"]
 
-[ext_resource type="PackedScene" path="res://src/gui_common/dialogs/CustomWindow.tscn" id="1"]
+[ext_resource type="PackedScene" uid="uid://du8sc8kjirguk" path="res://src/gui_common/dialogs/CustomWindow.tscn" id="1"]
 [ext_resource type="Script" uid="uid://dq7avc5py7nbb" path="res://src/awakening_stage/gui/SelectBuildingPopup.cs" id="2"]
 
-[node name="SelectBuildingPopup" type="Control"]
+[node name="SelectBuildingPopup" type="Control" node_paths=PackedStringArray("popup", "buttonsContainer", "cancelButton")]
 visible = false
 layout_mode = 3
 anchors_preset = 15
 anchor_right = 1.0
 anchor_bottom = 1.0
+grow_horizontal = 2
+grow_vertical = 2
 mouse_filter = 2
 script = ExtResource("2")
-PopupPath = NodePath("CustomWindow")
-ButtonsContainerPath = NodePath("CustomWindow/VBoxContainer/ScrollContainer/ButtonContainer")
-CancelButtonPath = NodePath("CustomWindow/VBoxContainer/Cancel")
+popup = NodePath("CustomWindow")
+buttonsContainer = NodePath("CustomWindow/VBoxContainer/ScrollContainer/ButtonContainer")
+cancelButton = NodePath("CustomWindow/VBoxContainer/Cancel")
 
 [node name="CustomWindow" parent="." instance=ExtResource("1")]
-anchors_preset = 0
```

Comment:
I'm slightly worried that these kind of anchor changes will break some GUI alignment in some of the prototypes, because they kind of relied on a broken setup where they want to have an invalid anchor preset. Fixing these properly would require tweaking the scenes these are instanced in and there re-selecting the right anchor types and repositioning the GUI element (as there they have a proper parent and Godot will allow setting the anchor value back to what it was).